scrawl_be/settings.py

```diff
@@ -107,7 +120,10 @@
     },
 ]
 
-
+CORS_ORIGIN_ALLOW_ALL = False
+CORS_ORIGIN_WHITELIST = [
+    'http://localhost:8000'
+]
```

Comment:
I assume we can just add on our Heroku FE when it's up and running?

Reply:
You got it!